charts/jaeger-operator/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: jaeger-operator Helm chart for Kubernetes
 name: jaeger-operator
-version: 2.19.1
+version: 2.19.2
```

small nit: 
```suggestion
version: 2.20.0
```

this is adding a "feature" and not fixing any bug

---

Updated 👍